core/data/stories.md

```diff
@@ -454,9 +546,81 @@
   - assessment_form
   - form{"name": "assessment_form"}
   - form{"name": null}
+  - slot{"self_assess_done": true}
   - slot{"symptoms": "none"}
   - utter_no_symptoms
   - utter_ask_when_first_symptoms
 * less
   - utter_self_isolate_symptom_free
   - utter_ask_anything_else
+* ask_question
+  - utter_can_help_with_questions
+  - question_answering_form
+  - form{"name": "question_answering_form"}
+  - form{"name": null}
+  - slot{"question_answering_status": "failure"}
+  - utter_question_answering_error
+  - utter_try_again_later
+  - utter_goodbye
+
+## QA - failure - no assessment after
+* greet
+  - utter_greet
+  - utter_ask_how_may_i_help
+* ask_question
+  - utter_can_help_with_questions
+  - question_answering_form
+  - form{"name": "question_answering_form"}
+  - form{"name": null}
+  - slot{"question_answering_status": "failure"}
+  - utter_question_answering_error
+  - utter_ask_assessment_after_error
```

Comment:
We have ask_assess elsewhere, we should be consistent when naming the utter actions.